lib/firmata.js

```diff
@@ -1148,13 +1177,13 @@ Board.prototype.sendI2CWriteRequest = function(slaveAddress, bytes) {
  * Write data to a register
  *
  * @param {number} address      The address of the I2C device.
- * @param {Array} cmdRegOrData  An array of bytes
+ * @param {array} cmdRegOrData  An array of bytes
```

Comment:
I thought there was some weird bug (maybe in jshint or jscs?) that required {Array} instead of {array}. Perhaps it's been fixed though. If you're not seeing any lint errors then I'm sure {array} is fine.

Reply:
Neither of these changes were intentional. Let me hop back into my commit and make sure I'm not submitting anything else I didn't mean to.